internal/view/browser.go

```diff
@@ -249,7 +249,7 @@ func (b *Browser) TableDataChanged(data *render.TableData) {
 
 	b.app.QueueUpdateDraw(func() {
 		b.refreshActions()
-		if !b.app.Config.K9s.UI.Reactive {
```

Comment:
I think this is actually correct. We only want to run this when not in reactive mode. Since if that mode is set the custom views watcher will take over.
Kind of spent here but I think this is correct?? Please double check me on this. 
Thanks Jayson!

Reply:
Hi, @derailed thanks for your quick reply, I'll start by getting this PR into a mergeable state.

Literally, if reactive mode is turned on, we should theoretically react to changes to views.yaml? I only went and found this place when I noticed the "CustomViews load failed" warning kept appearing in the upper right corner during use.

Reply:
@wjiec Thank you so much!! 
Please take a peek at my new pr. I think that's a better way to handle it when the configs are not there to start with?? 
Not really seeing straight at this juncture, so perhaps I am missing it? ;(

Reply:
@derailed Indeed, it's better to just create a config file if it doesn't exist

Reply:
Thank you Jason!